portfolio/src/main/webapp/css/master.css

```diff
@@ -0,0 +1,9 @@
+.project-card {
+  transition: 0.1s;
+}
+
+.project-card:hover {
+  box-shadow: 0px 0px 5px 1px rgba(232,232,232,1);
+  -moz-box-shadow: 2px 2px 5px 1px rgba(232,232,232,1);
+  -webkit-box-shadow: 2px 2px 5px 1px rgba(232,2
```

Comment:
Oops looks like we lost the end of this line :)

Reply:
How the heck did that happen lol. Fixing.